src/rendering/renderable.cpp

```diff
@@ -112,6 +113,22 @@ Renderable::Renderable(const ghoul::Dictionary& dictionary)
     dictionary.getValue(keyStart, _startTime);
     dictionary.getValue(keyEnd, _endTime);
 
+    std::string tagName;
+    ghoul::Dictionary tagNames;
+    if (dictionary.hasKeyAndValue<std::string>(KeyTag)) {
+        dictionary.getValue(KeyTag, tagName);
+        if (! tagName.empty())
+            addTag(tagName);
```

Comment:
Since we are not going to need it afterwards:  `addTag(std::move(tagName));` and save us the extra copy